app-crypt/adcli/adcli-0.9.2.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Tool for performing actions on an Active Directory domain"
+HOMEPAGE="https://www.freedesktop.org/software/realmd/adcli/adcli.html"
+SRC_URI="https://gitlab.freedesktop.org/realmd/adcli/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+DEPEND="
+	app-crypt/mit-krb5
+	net-nds/openldap:=[sasl]"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	doc? (
+		app-text/docbook-xml-dtd:4.3
+		app-text/xmlto
+		dev-libs/libxslt
+	)"
+
+src_prepare() {
+	default
+	eautoreconf
```

Comment:
Is this needed?

Reply:
Yes, the upstream code-base and the release tarballs do not contain things like configure-script and Makefile. Those need to be generated.